README.md

```diff
@@ -46,33 +38,37 @@
 - [WWDC](https://github.com/insidegui/WWDC) - The WWDC app. ![Open-Source Software][OSS Icon]
 - [Zsh](http://www.zsh.org/) -  A powerful command line shell. ![Open-Source Software][OSS Icon]
 
+### Design
+- [Sketch](http://bohemiancoding.com/sketch/) - An awesome vector editor like Adobe Illustrator. Sketch gives you the power, flexibility and speed you always wanted in a lightweight and easy-to-use package. Finally you can focus on what you do best: Design.
+- [PaintCode](http://www.paintcodeapp.com) - A powerful tool to turn your vector drawings into code.
```

Comment:
Again the description should not be that big, you should try and sum it up in 1 line, maybe something like `A Professional Digital Design Editor`?